helm/values-uat.yaml

```
@@ -0,0 +1,40 @@
+microservice-chart:
+  image:
+    repository: p4pauitncorecommonacr.azurecr.io/p4paauth-azure
```

ask to umberto if this name is right, or check the value of the Variable K8S_IMAGE_REPOSITORY_NAME inside the deploy pipeline (if it has been created inside Azure DevOps)
```suggestion
    repository: p4pauitncorecommonacr.azurecr.io/p4paauth
```

---

Antonio's suggestion is okay!